### PR TITLE
Add a sentence saying that @nbkhope is studying GitHub

### PR DESCRIPTION
**Pull request by nbkhope**
I went and edited the file test.txt and added a sentence with my handle and the topic I'm studying (GitHub).

To revert this change, either `git revert` with the commit SHA or just remove the sentence altogether.

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Hello there
-Lets try SSH
\ No newline at end of file
+Lets try SSH
```
Comment (nbkhope) [inline test.txt:2]: Could you add a blank line between these sentences?

**Review by nbkhope — COMMENTED** [review]
I left some comments for you to address

File: test.txt (hunk 1 above)
Comment (nbkhope) [reply test.txt:2]: Sounds good

**Comment by rendertolearn** [comment]
@nbkhope is studying GitHub!

**Review by rendertolearn — COMMENTED** [review]
Awesome job!

**Comment by rendertolearn** [comment]
Thank you for the great Demo lesson!

**Pull request merged**